generator/spring-boot/src/main/java/io/fabric8/maven/generator/springboot/SpringBootGenerator.java

```diff
@@ -63,9 +52,18 @@
     private static final String DEFAULT_SERVER_PORT = "8080";
 
     public enum Config implements Configs.Key {
-        color {{ d = "false"; }};
+        color {{
```

Comment:
sorry it was my bad when i did formatting IDEA split it as we.. wondering is there way to configure this style ;)